typescript/infra/config/environments/testnet4/funding.ts

```diff
@@ -28,7 +28,7 @@ export const keyFunderConfig: KeyFunderConfig<
   desiredBalancePerChain: {
     alfajores: '5',
     arbitrumsepolia: '0.1',
-    arcadiatestnet: '0.1',
+    // arcadiatestnet: '0.1',
```

Comment:
I don't totally recall but I think some infra scripts may create a provider for all supported chains and still fail even if we don't use the provider, but if this works thats fine

Reply:
`arcadiatestnet` been removed from the supported chains too - but I left the other references commented out for when we network stabilises again to add the deployment back